src/core/atomic.d

```diff
@@ -190,6 +254,14 @@ else version( AsmX86_32 )
         //
         // +=   -=  *=  /=  %=  ^^= &=
         // |=   ^=  <<= >>= >>>=    ~=
+        static if( op == "+=" && __traits(isIntegral, T) && is(T==V1) ) {
```

Comment:
Btw, the pragma wrote this in the console for the unittest:

ulong == ulong = true (op: +=)
ulongulong ==  == ulong = true (op: ulong-=)
 = true (op: +=)
ulong == ulong = true (op: -=)
int == int = true (op: +=)
int == int = true (op: -=)
ulong == ulong = true (op: +=)
ulong == ulong = true (op: -=)